Remove a test that will fail because we cannot garantuee that
these drivers are installed, especially on first installation.
--- test/dbi/tc_dbi.rb.orig	2010-06-15 19:58:58.732811127 +0200
+++ test/dbi/tc_dbi.rb	2010-06-15 19:59:10.004808935 +0200
@@ -139,18 +139,6 @@
        assert_respond_to(DBI, :connect)
    end
    
-   def test_available_drivers
-       assert_respond_to(DBI, :available_drivers)
-       assert_equal(
-       [
-            "dbi:Mysql:",
-            "dbi:ODBC:",
-            "dbi:Pg:",
-            "dbi:SQLite3:",
-            "dbi:SQLite:"
-       ], DBI.available_drivers.sort)
-   end
-
    # PRIVATE METHODS
    def test_parse_url
       assert_nothing_raised{ DBI.send(:parse_url, "dbi:foo:bar") }
